task-ballerina/src/task/listener.bal

```diff
@@ -167,14 +172,20 @@ isolated function validateConfiguration(TimerConfiguration|AppointmentConfigurat
 
             } else {
                 if (!(misfirePolicy is RecurringTaskPolicy)) {
-                    panic ListenerError("Wrong misfire policy has given for the repeating execution timer tasks.");
+                    panic ListenerError("Wrong misfire policy has been given for the repeating execution timer tasks.");
                 }
             }
         }
         if (configuration[INITIAL_DELAY] == ()) {
             configuration.initialDelayInMillis = configuration.intervalInMillis;
         }
     } else if (!(misfirePolicy is AppointmentMisfirePolicy)) {
-        panic ListenerError("Wrong misfire policy has given for the appointment task.");
+        panic ListenerError("Wrong misfire policy has been given for the appointment task.");
```

Comment:
Shall we say what you found but what is expected? Cant, we include the specific type in the specific configuration rather than separate configuration?